drivers/platform/maxim/max32650/maxim_i2c.c

```diff
@@ -40,6 +40,7 @@
 
 #include <stdint.h>
 #include <stdlib.h>
+#include <string.h>
```

Comment:
this change is unrelated to the commit. is this include required? same for all the other files changed in this commit.

Reply:
i think it is, probably for strcpy() usage

Reply:
or memcpy()

Reply:
Indeed, it's for memcpy(), but I'll change the commit message, since it's totally unrelated.